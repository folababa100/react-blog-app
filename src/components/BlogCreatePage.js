import React from 'react';
const BlogCreatePage = () => {
  return (
    <div>
      <p>Hello it me</p>
    </div>
  )
}

export default BlogCreatePage;
