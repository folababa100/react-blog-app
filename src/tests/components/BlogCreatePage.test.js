import React from 'react';
import { shallow } from "enzyme";
import BlogCreatePage from '../../components/BlogCreatePage';

test('BlogCreatePage to render correctly', () => {
  const wrapper = shallow(<BlogCreatePage/>);
  expect(wrapper).toMatchSnapshot();
})
