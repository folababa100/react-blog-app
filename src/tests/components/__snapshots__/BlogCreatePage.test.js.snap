// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogCreatePage to render correctly 1`] = `
<div>
  <p>
    Hello it me
  </p>
</div>
`;
